lib/model/query/audits.js

```diff
@@ -33,7 +33,7 @@ values (${actorId}, ${action}, ${acteeId}, ${(details == null) ? null : JSON.str
 // TODO: some sort of central table for all these things, not here.
 const actionCondition = (action) => {
   if (action === 'nonverbose')
-    return sql`action not in ('submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update', 'backup', 'config.set')`;
+    return sql`action not in ('submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update', 'backup')`;
```

Comment:
Now that there are more types of config and more details are logged for `config.set`, I think it makes sense to include `config.set` in the Frontend audit log.

Reply:
yes